src/schema/formats.js

```diff
@@ -103,6 +103,28 @@ export function isSecureUrl(value) {
   return ['https:', 'wss:'].includes(url.protocol);
 }
 
+export function isOrigin(value) {
+  let url;
+  try {
+    url = new URL(value);
+  } catch (e) {
```

Comment:
I don't know if our code supports it yet but theoretically we could use a catch without an error if we don't use it:

```suggestion
  } catch {
```